.github/workflows/deploy-grafana-dashboards.yaml

```diff
@@ -1,16 +1,24 @@
 name: Deploy grafana dashboards
 
-on: workflow_dispatch
+on:
+  schedule:
+    # Run at 05:00 every day, ref: https://crontab.guru/#0_5_*_*_*
+    - cron: "0 5 * * *"
+  workflow_dispatch:
 
 jobs:
   deploy_grafana_dashboards:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
     strategy:
       # Don't stop other deployments if one fails
       fail-fast: false
       matrix:
         include:
-          # The grafana for 2i2c cluster holds also info about all other clusters
+          # The grafana for 2i2c cluster holds also info about all other
+          # clusters and gets an additional dashboard deployed for that
+          - cluster_name: 2i2c
+            deploy_flags: --dashboards-dir=global-dashboards
```

Comment:
Because the `deployer grafana deploy-dashboards` command [wraps the `deploy` script](https://github.com/2i2c-org/infrastructure/blob/c168f862b0890916cd9cd1cd589c671b6371d2d5/deployer/commands/grafana/deploy_dashboards.py#L42-L49) of jupyterhub-grafana-dashboards I believe you also need to add support for this flag in the deployer, otherwise the flag won't be recognized.

https://github.com/2i2c-org/infrastructure/blob/c168f862b0890916cd9cd1cd589c671b6371d2d5/deployer/commands/grafana/deploy_dashboards.py#L13-L16

Reply:
Ah nice catch, resolved by 19e1ff2